README.rst

```diff
@@ -234,6 +234,9 @@ it should be easier to use in most cases.
     response.status is also set to HTTP error code.
 
   This option is set to True by default if you use SplashRequest.
+  Note that if you are using ``render.html`` endpoint, only response.url
```

Comment:
I think we can add 'headers' and 'cookies' to render.json, so it may be better to document how non-json endpoints behave and mention that render.json and /execute may not have all necessary keys/values in response